Exercises/4-count-types.js

```diff
@@ -1,5 +1,12 @@
 'use strict';
 
-const countTypesInArray = null;
+const countTypesInArray = array => {
+  const ret = {};
+  for (const a of array) {
```

Comment:
Try to give better names to identifiers, `value` or `item` or `element` instead of `a`.